### DISCUSSION TITLE
Arial font in linux machine

### DISCUSSION THREAD

**Discussion by inunezfe** [NREL/ReEDS-2.0#142] [Q&A: Features]
Hello ReEDS team

I was running the cases_small.csv file fixed and stumbled upon this warning:

<img width="1433" alt="image" src="https://user-images.githubusercontent.com/111766958/223146680-e8cb196b-4b58-4cb8-8962-2385605f9463.png">

Arial fonts are only available for Windows and MacOS based systems because microsoft office owns it, and install it in linux based system is complicated.

Just letting you know this may happen.

Thanks

**Comment by vcelloho** [comment]
Thanks for letting us know about this. Do the html reports generate in spite of the error with an alternative font or do the reports fail to generate?

**Reply by inunezfe** [reply]
I do not think they are getting generated, i just get a bunch of .csv files and a folder with some .png maps of the united states

![image](https://user-images.githubusercontent.com/111766958/223246893-588e17ca-dcf3-49c9-a610-6a2b1323f360.png)

![image](https://user-images.githubusercontent.com/111766958/223246988-f576ad6b-6314-4feb-947b-37a92cc5be6e.png)

**Reply by vcelloho** [reply]
I'm not sure what is causing this exact error for you. The code is supposed to be tolerant of Arial font not being available with an available font being automatically selected from the sanserif family of fonts. You should be able to change the code in postprocessing/plots.py to a font that you know to be available on your OS.

https://github.com/NREL/ReEDS_OpenAccess/blob/f681331ea0e87e690bf4d78f142e33ed35efc0e0/postprocessing/plots.py#L43-L47

https://github.com/NREL/ReEDS_OpenAccess/blob/f681331ea0e87e690bf4d78f142e33ed35efc0e0/postprocessing/plots.py#L871

https://github.com/NREL/ReEDS_OpenAccess/blob/f681331ea0e87e690bf4d78f142e33ed35efc0e0/postprocessing/plots.py#L890

https://github.com/NREL/ReEDS_OpenAccess/blob/f681331ea0e87e690bf4d78f142e33ed35efc0e0/postprocessing/plots.py#L935
